backend/temp/list.csv

```diff
@@ -0,0 +1,159 @@
+Github,Name,Telefon,Email,17.05.2017,24.05.2017,31.05.2017,07.06.2017,14.06.2017,21.06.2017,28.06.2017,05.07.2017,12.07.2017,19.07.2017,26.07.2017,02.08.2017,09.08.2017,16.08.2017,23.08.2017,30.08.2017,06.09.2017,13.09.2017,20.09.2017,27.09.2017,04.10.2017,11.10.2017,17.10.2017,25.10.2017,1.11.2017,8.11.2017,15.11.2017,22.11.2017,29.11.2017,6.12.2017,13.12.2017,20.12.2017,27.12.2017,3.1.2018,10.1.2018,17.1.2018,24.1.2018,31.1.2018,7.2.2017,14.2.2017,21.2.2017,28.2.2018,7.3.2018,14.3.2018,21.3.2018,28.3.2018,4.4.2018,11.4.2018,18.4.2018,25.4.2018,9.5.2018,16.5.2018,23.5.2018,30.5.2018,6.6.2018,13.6.2018,20.6.2018,27.6.2018,4.7.2018,11.7.2018,18.7.2018,25.7.2018,1.8.2018,8.8.2018,22.8.2018,29.8.2018,5.9.2018,12.9.2018,19.9.2018,26.9.2018,3.10.2018,10.10.2018,17.10.2018,24.10.2018,31.10.2018,7.11.2018,14.11.2018,21.11.2018,28.11.2018,5.12.2018,12.12.2018,19.12.2018,2.1.2019,9.9.2019,16.1.2018,23.1.2018,30.1.2019,6.2.2019,13.2.2019,20.2.2019,27.2.2019,6.3.2019,13.3.2019,20.3.2019,27.3.2019,3.4.2019,10.4.2019,17.4.2019,24.4.2019,8.5.2019,15.5.2019,22.5.2019,29.5.2019,5.6.2019,12.6.2019,19.6.2019,26.6.2019,3.7.2019,10.7.2019,17.7.2019,24.7.2019,31.7.2019,1.8.2019,7.8.2018,14.8.2019,21.8.2019,28.8.2019,4.9.2019,11.9.2019,18.9.2019,25.9.2019,2.10.2019,9.10.2019,23.10.2019,30.10.2019,6.11.2019,13.11.2019,20.11.2019,27.11.2019,4.12.2019,11.12.2019,18.12.2019,8.1.2020,15.1.2020,22.1.2020,29.1.2020,5.2.2020,12.2.2020,19.2.2020,26.2.2020,4.3.2020

```

Comment:
**big red flag here from me** - this should _not_ be here.

This file contains private information:

- we don't publish the real names of the members of organisation. Officially only few people are publicly listed as members of CodeForPoznan:
![image](https://user-images.githubusercontent.com/8676740/104512570-b0b28900-55ee-11eb-8851-9eba53231d41.png)
Bigger list of members is also accessible trough slack but that requires registration (thus, in a way - becoming a member).

- There are private mails and, in some cases, phone numbers. None of that should be publicly visible to people outside this organisation (which this PR breaks). Some people may choose to publish this information at their slack or github profile but there should _not_ be any intention in sharing that without explicit consent.

Also, this file is now baked in the git history, therefore removing it with another commit is in my opinion not enough.
Technically, it's even more cursed, because this commit is also on wistler's fork so to revert that file from public one would have to forcefully revert commit back (reset HEAD to previous commit).